### DISCUSSION TITLE
希望这次Docker能跑通吧 >_<

### DISCUSSION THREAD

**Discussion by JaHIY** [0-8-4/miui-auto-tasks#196] [展示和讲述 ｜Show and tell]
Ummmmm，我把`pdm.lock`删了，好像就能把Docker镜像制作出来了，就觉得很奇怪…

**Comment by 0-8-4** [comment]
总感觉不是很关事 我感觉是  `'%s\t%s\t%s\t%s\t%s\t%s\n' '0' '4' '*' '*' '*' '/usr/bin/env pdm run python /srv/miuitask.py';` 这个部份 
但是 我有看不出有什么问题 试试呗
 hotfix5 （

**Reply by 0-8-4** [reply]
似乎 解决了 还在 run

**Reply by JaHIY** [reply]
之前出错的地方是在`pdm install --prod`，我感觉是因为我用了国内镜像的原因，可能往`pdm.lock`里面写入了一些信息，导致安装失败。
反正没有`pdm.lock`的话，就`pdm`会自动去拉取最新版本的包，应该也没什么大问题。
